packages/medusa/src/loaders/index.ts

```diff
@@ -54,6 +55,8 @@ async function loadEntrypoints(
     next()
   })
 
+  await adminLoader({ app: expressApp, configModule })
```

Comment:
Let's load the admin at the end, after all of the backend had already loaded.

Reply:
Is there any reason for why you would prefer this? It's quite important that the middleware for serving the admin dev server does not get intercepted by another middleware, loading it before anything else makes sure that can't happen.

Reply:
@kasperkristensen simply because the BE API might not be ready by the time the admin has built and loaded

Reply:
Also we aren't really loading anything here (currently we are building the admin in prod, but once we have `medusa build` command we will do that there instead), we are just appending middleware and route handlers.